CHANGELOG.md

```diff
@@ -1,18 +1,21 @@
 ## CHANGELOG
 
-### 2.x.x (In progress)
+### 2.3.0 (202010-22)
```

Comment:
Looks like that date format is bit different

Reply:
Oh, thanks for spotting that. I made a typo 😓